custom-login/src/Home.jsx

```diff
@@ -0,0 +1,80 @@
+import { withAuth } from '@okta/okta-react';
+import React, { Component } from 'react';
+import { Container, Button, Header } from 'semantic-ui-react';
+import Navbar from './Navbar';
+import { checkAuthentication } from './helpers';
+
+export default withAuth(class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { authenticated: null, userinfo: null };
+    this.checkAuthentication = checkAuthentication.bind(this);
+  }
+
+  async componentDidMount() {
+    this.checkAuthentication();
+  }
+
+  async componentDidUpdate() {
+    this.checkAuthentication();
+  }
+
+  render() {
+    const resourceServerExamples = [
+      {
+        label: 'Node/Express Resource Server Example',
+        url: 'https://github.com/okta/samples-nodejs-express-4/tree/master/resource-server',
+      },
+      {
+        label: 'Java/Spring MVC Resource Server Example',
+        url: 'https://github.com/okta/samples-java-spring-mvc/tree/master/resource-server',
+      },
+    ];
+
+    return (
+      <div>
+        <Navbar />
+        {this.state.authenticated !== null &&
```

Comment:
Is this common React practice? Seems a bit funny to me to include the component directly inside of a conditional operand.

Reply:
Cool!